frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.scss

```diff
@@ -113,25 +127,36 @@
     }
   }
 
-  .simple-experiment-aliases-view {
-    height: 500px;
-    border: blue 1px solid;
+  // .simple-experiment-aliases-view {
```

Comment:
if it's really not needed lets just remove instead of comment out

Reply:
Yes, I guess that's the code you've used temporarily before creating a separate component right? I just wanted to make sure it's no longer needed. I will remove it and commit again.

Reply:
yep i don't see that selector used anywhere, looks like something I left in there unused